install.md

```diff
@@ -0,0 +1,496 @@
+# Setup instructions for workflows created with the DataJoint Elements
+
++ The following document describes the steps to setup a development environment 
+so that you can use the DataJoint Elements to build and run a workflow on your 
+local machine.
+
++ The DataJoint Elements can be combined together to create a workflow that 
+matches your experimental setup.  We have created example workflows (e.g. 
+`workflow-array-ephys`, `workflow-calcium-imaging`) for your reference.  In this
+ tutorial we will install these example DataJoint workflows.
+ 
++ These instructions can be adapted for your custom DataJoint workflow.
+
++ There are several ways to create a development environment.  Here we will 
+discuss one method in detail, and will highlight other methods along the way.  
+If you have already set up certain components, feel free to skip those steps.
+
++ You will need administrative privileges on your system for the following setup
+instructions.
+
+## System architecture
+![Architecture](/images/install_architecture.jpeg)
```

Comment:
For diagrams, use PNG, not JPEG. JPEG compression is designed for photography and introduces artifacts or inefficiencies for diagrams with sharp and flat features.